tests/test_viz/test_wiggle.py

```
@@ -167,3 +177,91 @@ def test_3d_patch_raises(self, range_patch_3d):
+        offsets = [traces[i][0, 1] - patch.data[i, 0] for i in shown]
+        assert np.allclose(ticks, offsets)
+
+    def test_2d_shade(self, small_patch):
+        """
+        Shading makes one polygon per trace which covers only the part of the
+        trace above its offset.
+        """
+        # Center the traces so each has both signs.
+        patch = small_patch - np.mean(small_patch.data)
+        ax = patch.viz.wiggle(shade=True)
+        poly, verts = _get_shade_vertices(ax)
```

Ruff (0.16.1)

[warning] Unpacked variable `verts` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)